src/server/common/model/section/license.test.js

```diff
@@ -9,16 +12,45 @@ describe('License', () => {
   })
 
   describe('validate', () => {
-    it('should return invalid', () => {
-      const result = License.fromState().validate()
+    it('should return valid if all nested objects are valid', () => {
+      const licenseData = {
+        emailAddress: testEmail
+      }
+      const result = License.fromState(licenseData).validate()
+
+      expect(result.isValid).toBe(true)
+      expect(result.errors).toEqual({})
+    })
+
+    it('should return invalid if any nested object is invalid', () => {
+      const licenseData = {
+        emailAddress: ''
+      }
+
+      const result = License.fromState(licenseData).validate()
+
       expect(result.isValid).toBe(false)
+      expect(result.errors).toHaveProperty('license')
```

Comment:
ultimately, we're going to want this to be the key of the question that's erroring: 

```javascript
{
  emailAddress: { text: "...." }
}
```

In more complex sections, that will give us the data we need to know which question route users to.

Reply:
Done